SUPPORT.md

```diff
@@ -1,55 +1,19 @@
 OpenSSL User Support resources
 ==============================
 
-_Under Construction; not more than a collection of text fragments yet._
-
 See the OpenSSL website www.openssl.org for details on how to obtain
-commercial technical support. Free community support is available through the
-openssl-users email list (see
-<https://www.openssl.org/community/mailinglists.html for> further details).
-
-If you have any problems with OpenSSL then please take the following steps
-first:
-
- - Download the latest version from the repository
-   to see if the problem has already been addressed
- - Configure with no-asm
- - Remove compiler optimization flags
-
-If you wish to report a bug then please include the following information
-and create an issue on GitHub:
-
- - OpenSSL version: output of 'openssl version -a'
- - Configuration data: output of 'perl configdata.pm ==dump'
- - OS Name, Version, Hardware platform
- - Compiler Details (name, version)
- - Application Details (name, version)
- - Problem Description (steps that will reproduce the problem, if known)
- - Stack Traceback (if the application dumps core)
-
-Just because something doesn't work the way you expect does not mean it
-is necessarily a bug in OpenSSL. Use the openssl-users email list for this type
-of query.
-
-For *questions* on how to use OpenSSL or what went wrong when you
-tried something, our primary resource is the mailing list
-openssl-users@openssl.org, where you can get help from others in the
-OpenSSL community (which includes the developers as time permits).
-
-Only subscribers can post to openssl-users@openssl.org (although the
-archives are public).
-For more information, see <https://www.openssl.org/community/mailinglists.html>
+commercial technical support.
```

Comment:
Perhaps a more precise link to https://www.openssl.org/support/contracts.html at this point?

Reply:
Sure.

Reply:
Done